### PR TITLE
Feb 11, 2021 Release Notes

### PR DESCRIPTION
**Pull request by huangjeff5**

### CODE REVIEW

**Comment by huangjeff5** [comment]
@jhuleatt I don't know what the linter is complaining about... the error is not very clear. Could you take a look?

File: firestore-send-email/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## Version 0.1.8
+
+fixed - Removed code coverage check on ci
```
Comment (kevinthecheung) [inline firestore-send-email/CHANGELOG.md:3]: CI?

File: storage-resize-images/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+## Version 0.1.18
+
+fixed - capitalised file extensions (#549)
```
Comment (kevinthecheung) [inline storage-resize-images/CHANGELOG.md:3]:
```suggestion
fixed - capitalized file extensions (#549)
```

(sorry, UK)

**Review by kevinthecheung — APPROVED** [review]

File: storage-resize-images/CHANGELOG.md (hunk 2 above)
Comment (jhuleatt) [reply storage-resize-images/CHANGELOG.md:3]: How about:

```
fixed - fixed a bug that caused resized images to be named incorrectly if their file extension was capitalized (#549)
```

File: storage-resize-images/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,9 @@
...
+
+fixed - Removed code coverage check on ci
```
Comment (jhuleatt) [inline storage-resize-images/CHANGELOG.md:5]: please remove this (this is a repo maintenance detail, not related to the functionality of the extension)

File: firestore-send-email/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,7 @@
...
+
```
Comment (huangjeff5) [inline firestore-send-email/CHANGELOG.md:4]: @jhuleatt 
If we remove the ci change, then there are no changes. How do we want to represent that here?

File: firestore-send-email/CHANGELOG.md (hunk 1 above)
Comment (jhuleatt) [reply firestore-send-email/CHANGELOG.md:3]: how about:

```
docs: updated postinstall instructions to clarify which ports are recommended for SMTP connections
```

File: firestore-send-email/CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,7 @@
+## Version 0.1.8
+
+docs - updated postinstall instructions to clarify which ports are recommended for SMTP connections (#582)
```
Comment (jhuleatt) [inline firestore-send-email/CHANGELOG.md:3]:
```suggestion
docs - updated "SMTP connection URI" parameter description to clarify which ports are recommended for SMTP connections (#582)
```

**Review by jhuleatt — APPROVED** [review]

**Pull request merged**